pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -502,3 +510,74 @@ func checkForPodSecondaryNetworkAttachement(pod *corev1.Pod) (string, bool) {
 		return netObj, false
 	}
 }
+
+// reconcileSecondaryInterfaces restore interfacestore and cniCache when agent restart.
+func (pc *PodController) reconcileSecondaryInterfaces(pIfacestore interfacestore.InterfaceStore) error {
+	err := pc.initializeInterfaceStore()
```

Comment:
Maybe better to move this call to NewPodController(), so it initializes the interface store first, and then calls reconcileSecondaryInterfaces to reconcile interfaces in the store?